src/Nethermind/Nethermind.Core/Crypto/Keccak.cs

```diff
@@ -56,16 +70,72 @@ public static ValueHash256 Compute(ReadOnlySpan<byte> input)
                 return OfAnEmptyString;
             }
 
-            Unsafe.SkipInit(out ValueHash256 keccak);
-            KeccakHash.ComputeHashBytesToSpan(input, MemoryMarshal.AsBytes(MemoryMarshal.CreateSpan(ref keccak, 1)));
-            return keccak;
+            Unsafe.SkipInit(out ValueHash256 hash);
+            Unsafe.SkipInit(out BytesWrapper cacheKey);
+            if (input.Length <= 32)
```

Comment:
should we consider length > 32? Maybe put 32 to a const so it can be easily adjusted.

Reply:
Sources of > 32 more likely to be single use? e.g. transactions, receipts, changed nodes rlps in trie
Also more expensive to store larger items as keys

Added const